frontend/src/pages/LoginPage.tsx

```diff
@@ -25,16 +25,16 @@ export default function LoginPage() {
   // Hits the token endpoint, and stores the token in local storage. Displays incorrect password text if error returned from endpoint
   const onSubmit = (event: FormEvent<HTMLFormElement>): void => {
     // Make the request, and show error message if password is wrong
-    AUTH_API.login(password)
+    const req: LoginReq = {
+      password: password,
+    };
+
+    AUTH_API.login(req)
       .then((response) => {
-        if (response.data.access) {
-          setAuthToken(response.data.access);
-          sessionStorage.setItem("refreshToken", response.data.refresh);
-          sessionStorage.setItem("accessToken", response.data.access);
-          navigate("/books");
-        } else {
-          delete axios.defaults.headers.common["Authorization"];
```

Comment:
This looked like some error checking I don't know if it may be needed anymore because the functionality exists somewhere else, or was it redundant code.

Reply:
This code was not actually being used, because we have a specific instance of axios being used in the src/apis/Config.tsx file (export const API), for which this code had no impact.